includes/gateways/class-wc-subscriptions-gateway-restrictions-manager.php

```diff
@@ -46,6 +46,24 @@ public static function enqueue_scripts() {
 				'decimal_point_separator'      => $decimal_separator,
 			);
 
+			/**
+			 * Allow WC Payments to return the minimum amount that can be processed.
+			 *
+			 * @since 1.5.0
+			 *
+			 * @param bool|float The minimum amount that can be processed in the given currency.
+			 * @param string.    The currency.
+			 */
+			$minimum_processable_amount = apply_filters( 'woocommerce_subscriptions_minimum_processable_recurring_amount', false, get_woocommerce_currency() );
+
+			if ( $minimum_processable_amount ) {
```

Comment:
If you filter the value as a random string `"helloworld"` you get a warning:

```
PHP Warning:  number_format() expects parameter 1 to be float, string given in /Users/matt/local/woo/wp-content/plugins/woocommerce-subscriptions-core/includes/gateways/class-wc-subscriptions-gateway-restrictions-manager.php on line 60
```

```suggestion
			if ( $minimum_processable_amount && is_numeric( $minimum_processable_amount ) ) {
```

Reply:
I've updated to only check if the amount is numeric.